platformio.ini

```diff
@@ -82,6 +82,7 @@ extra_configs =
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 [libraries]
+arduino_stl = ciband/avr_stl @ ^1.2.3
```

Comment:
For Zgateway BT White and blacklist mechanism we use:
```c
`stl = https://github.com/mike-matera/ArduinoSTL.git#7411816`
```
The one that you are proposing seems more maintained. Would it be possible to do the replacement into platformio.ini so as to use only yours?

Reply:
Sure.